mypy/main.py

```diff
@@ -245,6 +245,7 @@ def add_invertible_flag(flag: str,
                         version='%(prog)s ' + __version__)
     parser.add_argument('--python-version', type=parse_version, metavar='x.y',
                         help='use Python x.y')
+    parser.add_argument('--python', action='store', help="Point to a Python executable.")
```

Comment:
This is rather laconic. Maybe "Python executable whose installed packages will be used in typechecking".